db/merge_helper.cc

```diff
@@ -275,10 +279,6 @@ Status MergeHelper::MergeUntil(InternalIterator* iter,
     // We haven't seen the beginning of the key nor a Put/Delete.
     // Attempt to use the user's associative merge function to
     // merge the stacked merge operands into a single operand.
-    //
-    // TODO(noetzli) The docblock of MergeUntil suggests that a successful
```

Comment:
Seems clear to me that MergeInProgress is the right thing to return. But it's more of a philosophical question since none of the callers of MergeUntil distinguish between OK and MergeInProgress.